internal/report/source.go

```diff
@@ -239,7 +239,8 @@ func sourceCoordinates(asm map[int][]assemblyInstruction) (start, end int) {
 // assemblyPerSourceLine disassembles the binary containing a symbol
 // and classifies the assembly instructions according to its
 // corresponding source line, annotating them with a set of samples.
-func assemblyPerSourceLine(objSyms []*objSymbol, rs graph.Nodes, src string, obj plugin.ObjTool) map[int][]assemblyInstruction {
+func assemblyPerSourceLine(objSyms []*objSymbol, rs graph.Nodes,
+	src string, obj plugin.ObjTool, intelSyntax bool) map[int][]assemblyInstruction {
```

Comment:
Please avoid unrelated formatting changes. Keep this one line, long lines are fine in Go.

Reply:
Ack. Now fixed.